backend/app.py

```diff
@@ -1,10 +1,28 @@
 from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+import os
 
-app = Flask(__name__)
+# Create SQLAlchemy object
+db = SQLAlchemy()
 
-@app.route('/healthz')
-def healthz():
-    return '', 200
+def create_app():
+    app = Flask(__name__)
+
+    # Configure the SQLite database path
+    instance_path = os.path.expanduser('~/td2stats_instance')
```

Comment:
Normally you should place the database file in the directory of the repo. That way it will work on anyone's machine.

Reply:
You're talking specifically about my instance folder contents, or the create_db.py file?

Reply:
The app.db file. And then you don't need this folder.

Reply:
![Screenshot 2024-10-30 222359](https://github.com/user-attachments/assets/53ba33d3-d037-41b3-b75c-478e18d9f691)